seqr/management/tests/check_bam_cram_paths_2_3_tests.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import unicode_literals
+
+import mock
+
+from django.core.management import call_command
+from django.test import TestCase
+
+
+class CheckBamCramPathsTest(TestCase):
+    fixtures = ['users', '1kg_project']
+
+    @mock.patch('seqr.management.commands.check_bam_cram_paths.logger')
+    @mock.patch('seqr.views.utils.dataset_utils.validate_alignment_dataset_path')
+    def test_command(self, mock_validate_path, mock_logger):
```

Comment:
Codacy found an issue: [Method could be a function](https://app.codacy.com/manual/hanars/seqr/pullRequest?prid=5795824)

Reply:
not relevant